frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -449,6 +449,9 @@ class Worksheet extends React.Component {
         if (item && (!item.props || !item.props.item)) {
             // Skip "no search results" items and scroll past them.
             const offset = index - this.state.focusIndex;
+            if (offset === 0) {
```

Comment:
In what scenario did this give you an error?

Reply:
I was hitting on the last row of a search result table, clicked edit source and save